app/models/miq_report/import_export.rb

```diff
@@ -107,6 +107,8 @@ def import_from_hash(report, options = nil)
     # @param cache [Hash] cache that holds yaml for the views
     def load_from_view_options(db, current_user = nil, options = {}, cache = {})
       filename = MiqReport.view_yaml_filename(db, current_user, options)
+      raise Errno::ENOENT, "Unable to find view yaml file for db #{db.inspect}" if filename.nil?
```

Comment:
I'm curious if filename can be non-nil but still not exist on the filesystem. Should we do `if (filename.nil || !File.exist?(filename))`